save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AwesomeBenchmarksView.kt

```diff
@@ -96,6 +98,21 @@ class AwesomeBenchmarksView : AbstractView<PropsWithChildren, AwesomeBenchmarksS
         }
     }
 
+    override fun componentDidMount() {
+        super.componentDidMount()
+
+        val href = window.location.href
+        val tab = if (href.contains(Regex("/archive/[^/]+/[^/]+"))) {
+            href.substringAfterLast(URL_PATH_DELIMITER).run { BenchmarkCategoryEnum.values().find { it.name.lowercase() == this } }
```

Comment:
Suggest to extract complicated lines of code to a separate function, maybe to companion object of enum

Reply:
Done